services/npm/npm-last-update.tester.js

```diff
@@ -3,51 +3,79 @@ import { createServiceTester } from '../tester.js'
 
 export const t = await createServiceTester()
 
-t.create('last updated date (valid package)')
+t.create('last updated date, no tag, valid package')
   .get('/verdaccio.json')
   .expectBadge({
     label: 'last updated',
     message: isFormattedDate,
   })
 
-t.create('last updated date (invalid package)')
+t.create('last updated date, no tag, invalid package')
   .get('/not-a-package.json')
   .expectBadge({
     label: 'last updated',
     message: 'package not found',
   })
 
-t.create('last update from custom repository (valid scenario)')
+t.create('last updated date, no tag, custom repository, valid package')
   .get('/verdaccio.json?registry_uri=https://registry.npmjs.com')
   .expectBadge({
     label: 'last updated',
     message: isFormattedDate,
   })
 
-t.create('last update scoped package (valid scenario)')
+t.create('last updated date, no tag, valid package with scope')
   .get('/@npm/types.json')
   .expectBadge({
     label: 'last updated',
     message: isFormattedDate,
   })
 
-t.create('last update scoped package (invalid scenario)')
+t.create('last updated date, no tag, invalid package with scope')
   .get('/@not-a-scoped-package/not-a-valid-package.json')
   .expectBadge({
     label: 'last updated',
     message: 'package not found',
   })
 
-t.create('last updated date with tag (valid scenario)')
+t.create('last updated date, with tag, valid package')
   .get('/verdaccio/latest.json')
   .expectBadge({
     label: 'last updated',
     message: isFormattedDate,
   })
 
-t.create('last updated date (invalid tag)')
+t.create('last updated date, with tag, invalid package')
+  .get('/not-a-package/doesnt-matter.json')
+  .expectBadge({
+    label: 'last updated',
+    message: 'package not found',
+  })
+
+t.create('last updated date, with tag, invalid tag')
   .get('/verdaccio/not-a-valid-tag.json')
   .expectBadge({
     label: 'last updated',
     message: 'tag not found',
   })
+
+t.create('last updated date, with tag, custom repository, valid package')
+  .get('/verdaccio/latest.json?registry_uri=https://registry.npmjs.com')
+  .expectBadge({
+    label: 'last updated',
+    message: isFormattedDate,
+  })
+
+t.create('last updated date, with tag, valid package with scope')
+  .get('/@npm/types/latest.json')
+  .expectBadge({
+    label: 'last updated',
+    message: isFormattedDate,
+  })
+
+t.create('last updated date, with tag, invalid package with scope')
+  .get('/@not-a-scoped-package/not-a-valid-package/doesnt-matter.json')
+  .expectBadge({
+    label: 'last updated',
+    message: 'package not found',
+  })
```

Comment:
Because I've split with/without tags into 2 classes, I've doubled up the test suite so we're testing both classes for each scenario.